documentation/api/io.strimzi.api.kafka.model.template.ContainerTemplate.adoc

```diff
@@ -0,0 +1,25 @@
+You can set custom security context and environment variables for a container.
+
+The environment variables are defined under the `env` property as a list of objects with `name` and `value` fields.
+The following example shows two custom environment variables set for the Kafka broker containers:
+
+[source,yaml,subs=attributes+]
+----
+# ...
+kind: Kafka
+spec:
+    kafka:
+        template:
+            kafkaContainer:
+                env:
+                    - name: TEST_ENV_1
```

Comment:
Could we swap _test_ for _example_?  We should probably keep test-y things out of the doc